extensions/arc/package.json

```diff
@@ -1130,7 +1130,7 @@
     "@types/sinon": "^9.0.4",
     "@types/uuid": "^8.3.0",
     "@types/yamljs": "^0.2.31",
-    "@microsoft/azdata-test": "^1.4.0",
+    "@microsoft/azdata-test": "file:C:\\azdata-test",
```

Comment:
@Charles-Gagnon  This requires a new version of azdata-test to include https://github.com/microsoft/azdata-test/pull/2.

Reply:
https://www.npmjs.com/package/@microsoft/azdata-test/v/1.5.0